plugins/inputs/bond/bond.go

```diff
@@ -14,24 +14,44 @@ import (
 
 // default host proc path
 const defaultHostProc = "/proc"
+const defaultHostSys = "/sys"
 
 // env host proc variable name
 const envProc = "HOST_PROC"
+const envSys = "HOST_SYS"
 
 type Bond struct {
 	HostProc       string   `toml:"host_proc"`
+	HostSys        string   `toml:"host_sys"`
+	SysDetails     bool     `toml:"collect_sys_details"`
 	BondInterfaces []string `toml:"bond_interfaces"`
+	BondType       string
+}
+
+type sysFiles struct {
+	ModeFile    string
+	SlaveFile   string
+	ADPortsFile string
 }
 
 var sampleConfig = `
```

Comment:
Please make this a const
```suggestion
const sampleConfig = `
```